<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReserverApp</class>
 <widget class="QWidget" name="ReserverApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>source:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="source">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="source_view_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>view</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dest_view_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>view</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>dest:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copy_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QLabel" name="weight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>weight:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
